runtime/laos/src/lib.rs

```diff
@@ -127,6 +128,10 @@ construct_runtime!(
 		CumulusXcm: cumulus_pallet_xcm = 32,
 		MessageQueue: pallet_message_queue = 34,
 
+		// Governance
+		Council: pallet_collective::<Instance1> = 40,
```

Comment:
what about renaming `Instance1` to something more meaningful? `CouncilCollective` could be an option

Reply:
I't not something you can decide